server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryManifestAttributes.java

```diff
@@ -0,0 +1,43 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+/**
+ provides attribute-names for the cloud foundry manifest
+ */
+public enum CloudFoundryManifestAttribute {
```

Comment:
The tests are failing because you only renamed the class, please rename the whole file and run the tests locally before you push.

Reply:
ah fuck...really stupid